README.md

```diff
@@ -113,10 +113,14 @@ On Starknet, the deployment process is in two steps:
     ```bash
     make declare
     ```
-    Copy the declare Class Hash provided to use in the following step.
+    Copy the declared hashes from the Factory and the Pool for the following step.
 3. Deploy:
-   ```bash
-   make deploy CLASS_HASH="<CLASS_HASH>"
+   ```bash"
+   // FACTORY_CLASS_HASH: Include the class hash of the YASFactory declare.
+   // OWNER_ADDRESS: Include a Starknet address.
+   // POOL_CLASS_HASH: Include the class hash of the YASPool declare.
+
+   make deploy CLASS_HASH="<FACTORY_CLASS_HASH>" CTOR_ARGS="<OWNER_ADDRESS> <POOL_CLASS_HASH>"
```

Comment:
Are we sure that it works with the semi colons? Specially for the case of `CTOR_ARGS` with two arguments and a space in the middle.

Reply:
Yes, initially it was without the quotation marks, but when we added the second parameter "pool_class_hash," they became mandatory; otherwise, it doesn't work :(. This could be something specific for the Makefile because it's not necessary with starkli.